src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js

```diff
@@ -28,6 +28,33 @@ function isClusterAvailable(cluster) {
   return ready;
 }
 
+function getClusterLastCheckedIn(cluster) {
+  let ready = false;
+  if (
+    cluster?.spec.clusterData &&
+    cluster?.spec.clusterData.cluster_status &&
+    cluster?.spec.clusterData.cluster_status.conditions &&
+    cluster?.spec.clusterData.cluster_status.conditions.length > 0
+  ) {
+    for (
+      let index = 0;
+      index < cluster?.spec.clusterData?.cluster_status?.conditions?.length;
```

Comment:
We can probably access without  `?.` as we have already validated that it exists above.

Reply:
yeah, removed that and added this before for loop.